project/BuildSettings.scala

```diff
@@ -28,14 +31,32 @@ object BuildSettings {
       fork in Test := true   // Sometimes this causes sbt test to fail,
     )
 
-  def projectSettings(assembly: Boolean = false) = buildSettings ++
-    Seq(resolvers += Classpaths.typesafeReleases,
-        resolvers ++= socrataResolvers,
-        scalacOptions ++= Seq("-Xlint", "-deprecation", "-Xfatal-warnings", "-feature"))
+  def projectSettings = buildSettings ++
+    Seq(
+      resolvers += Classpaths.typesafeReleases,
+      resolvers ++= socrataResolvers,
+      scalacOptions ++= Seq("-Xlint", "-deprecation", "-Xfatal-warnings", "-feature")
+    )
+
+  lazy val socrataResolvers =
+    Seq(
+      "Open Source Geospatial Foundation Repository" at "http://download.osgeo.org/webdav/geotools",
+      "spray repo" at "http://repo.spray.io",
+      "velvia maven" at "https://dl.bintray.com/velvia/maven"
+    )
+
 
-  lazy val socrataResolvers = Seq(
-    "Open Source Geospatial Foundation Repository" at "http://download.osgeo.org/webdav/geotools",
-    "spray repo" at "http://repo.spray.io",
-    "velvia maven" at "https://dl.bintray.com/velvia/maven"
-  )
+  lazy val rootSettings =
+    Seq(
+      // do not publish (no geospace folder created)
+      publish := {},
+      // do not publish locally
+      publishLocal := {},
```

Comment:
So I wanted to configure the parent project so it doesnt build artifacts. I originally placed the rootSettings (hopefully clearly named?) in the build.sbt file but felt that it belongs here instead. As you can see in the build.sbt that only should affect the root project and nothing else.